stable/mlrun/templates/db-configmap.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if eq .Values.httpDB.sqlDBType "mysql" }}
+{{- if .Values.rbac.create }}
```

Comment:
I know I put it in my PR as well, and I guess it's here since you just copy pasted, looks like not relevant